snuba/web/db_query.py

```diff
@@ -504,7 +504,7 @@ def _raw_query(
     timer: Timer,
     # NOTE: This variable is a piece of state which is updated and used outside this function
     stats: MutableMapping[str, Any],
-    trace_id: Optional[str] = None,
+    trace_id: str = str(uuid.uuid4()),
```

Comment:
why do you need this? also this will make it so the uuid that is calculated at import time is the default argument for every subsequent call.

Reply:
If the `trace_id` isn't passed in here, I think we should populate it with a default non-null `trace_id`, so that if an error occurs and the QueryException is written to querylog, we won't have a null `trace_id`

Reply:
ok let's do the default all zeroes one by default then to cover our bases